### ISSUE TITLE
make the view more configurable

### ISSUE THREAD

**Issue by jwaldmann** [haskellnews/haskellnews#58]
We currently have "mixed" (default) and "grouped" views. 

The thing would be much more usable (for me) if I could

* add some more mailing lists (libraries, ghc-devs)
* remove  homework questions (SO) and noise (twitter, github)

Perhaps extra parameters for the route, like `/grouped/<source1>/<source2>/...` where
`sourceN` are identifiers for the feed sources (as they are already being used internally).

**Comment by lwm** [comment]
> add some more mailing lists (libraries, ghc-devs)

Sure, go for it!

> remove homework questions (SO) and noise (twitter, github)

I'd like to filter/improve this but I am not sure removing it is useful.

> Perhaps extra parameters for the route, like /grouped/source1/source2/... where
sourceN are identifiers for the feed sources (as they are already being used internally).

Kinda reminded me of https://github.com/haskellnews/haskellnews/issues/15 but you want to choose which group of sources you will view?

**Comment by jwaldmann** [comment]
Yes: want to chwant to choose which group of sources you will view.

**Comment by jwaldmann** [comment]
I have a working version at 9c24ab247c025a988f250351e696f7c4529221c5 .
This lets me view `http://localhost:10010/grouped/HaskellCafe+Reddit` etc.
